### PR TITLE
New docs website - Scaffolding - Create `website` project folder

### PR DESCRIPTION
**Pull request by didoo**
### :pushpin: Summary

This is the initial scaffolding for the `Field-Guide` Ember app.

The integration with CI and Vercel deploys will be done in follow up tickets.

### :hammer_and_wrench: Detailed description

In this PR I have:
- created the `website` Ember app
  - the `website` folder has been added to the workspaces configuration
- installed `empress/field-guide` in the `website` application

### :link: External links

- Jira Epic: https://hashicorp.atlassian.net/browse/HDS-625
- Jira Issue: https://hashicorp.atlassian.net/browse/HDS-702

***

### 👀 How to review

👉 Review commit-by-commit or by files changed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    "ember-data": "~4.3.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^8.1.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-page-title": "^7.0.0",
+    "ember-qunit": "^5.1.5",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "~4.3.0",
+    "ember-template-lint": "^4.3.0",
+    "ember-welcome-page": "^6.2.0",
```
Comment (Dhaulagiri) [inline website/package.json:51]: this can be removed

File: website/package.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+  "name": "website",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for website goes here",
+  "repository": "",
```
Comment (Dhaulagiri) [inline website/package.json:6]:
```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/hashicorp/design-system.git",
    "directory": "website"
  },
```

File: website/package.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  "license": "MIT",
```
Comment (Dhaulagiri) [inline website/package.json:7]:
```suggestion
  "license": "MPL-2.0",
```

File: website/package.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  "author": "",
```
Comment (Dhaulagiri) [inline website/package.json:8]:
```suggestion
  "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
```

**Review by Dhaulagiri — COMMENTED** [review]
Made a couple of minor suggestions.

We'll also want to remove `website/package-lock.json` file that was created (tip: if you pass `--yarn` when creating an ember project it'll use yarn instead of npm)

**Comment by didoo** [comment]
@Dhaulagiri fixed all the issues. can you re-review? thanks

**Review by Dhaulagiri — APPROVED** [review]

**Pull request merged**